docs/core/diagnostics/runtime-loader-binder-events.md

```diff
@@ -0,0 +1,337 @@
+|-----------------------------------|-----------|-----------|
+|`LoaderKeyword` (0x8)|`DomainModuleLoad_V1`|Informational (4)|
+
+|Event|Event ID|Description|
+|-----------|--------------|-----------------|
+|`AssemblyDCStart_V1`|155|Enumerates assemblies during a start rundown.|
+
+
+|Field name|Data type|Description|
+|----------------|---------------|-----------------|
+|AssemblyID|win:UInt64|Unique ID for the assembly.|
+|AppDomainID|win:UInt64|ID of the domain of this assembly.|
+|BindingID|win:UInt64|ID that uniquely identifies the assembly binding.|
+|AssemblyFlags|win:UInt32|0x1: Domain neutral assembly.<br /><br /> 0x2: Dynamic assembly.<br /><br /> 0x4: Assembly has a native image.<br /><br /> 0x8: Collectible assembly.|
+|AssemblyName|win:UnicodeString|Fully qualified assembly name.|
+|ClrInstanceID|win:UInt16|Unique ID for the instance of CoreCLR.|
+
+
+## AssemblyLoadStart Event
+
+|Keyword for raising the event|Event|Level|
+|-----------------------------------|-----------|-----------|
+|`Binder` (0x4)|`AssemblyLoadStart`|Informational (4)|
+
+|Event|Event ID|Description|
+|-----------|--------------|-----------------|
+|`AssemblyLoadStart`|290|An assembly load has been requested.
+
+|Field name|Data type|Description|
+|----------------|---------------|-----------------|
+|AssemblyName|win:UnicodeString|Name of assembly name.|
+|AssemblyPath|win:UnicodeString|Path of assembly name.|
+|RequestingAssembly|win:UnicodeString|Name of the requesting ("parent") assemb.|
+|AssemblyLoadContext|win:UnicodeString|Load context of the assembly.|
+|RequestingAssemblyLoadContext|win:UnicodeString|Load context of the requesting ("parent") assembly.|
+|ClrInstanceID|win:UInt16|Unique ID for the instance of CoreCLR.|
+
+
+## AssemblyLoadStop Event
+
+|Keyword for raising the event|Event|Level|
+|-----------------------------------|-----------|-----------|
+|`Binder` (0x4)|`AssemblyLoadStart`|Informational (4)|
+
+|Event|Event ID|Description|
+|-----------|--------------|-----------------|
+|`AssemblyLoadStart`|291|An assembly load has been requested.
+
+
+|Field name|Data type|Description|
+|----------------|---------------|-----------------|
+|AssemblyName|win:UnicodeString|Name of assembly name.|
+|AssemblyPath|win:UnicodeString|Path of assembly name.|
+|RequestingAssembly|win:UnicodeString|Name of the requesting ("parent") assemb.|
+|AssemblyLoadContext|win:UnicodeString|Load context of the assembly.|
+|RequestingAssemblyLoadContext|win:UnicodeString|Load context of the requesting ("parent") assembly.|
+|Success|win:Boolean|Whether the assembly load succeeded.|
+|ResultAssemblyName|win:UnicodeString|The name of assembly that was loaded.|
+|ResultAssemblyPath|win:UnicodeString|The path of the assembly that was loaded from.|
+|Cached|win:UnicodeString|Whether the load was cached.|
+|ClrInstanceID|win:UInt16|Unique ID for the instance of CoreCLR.|
+
+## ResolutionAttempted Event
+
+|Keyword for raising the event|Level|
```

Comment:
```suggestion
|Keyword for raising the event|Event|Level|
```
More tables that didn't print. You can take a pass over the PR in rich diff mode and any remaining ones should stick out.